doc/releasing.md

```diff
@@ -0,0 +1,60 @@
+# Releasing
+
+This document is aimed at Maintainers and describes how to release a new version of the packages contained in this repository.
+We aim to eventually automate this process as much as possible.
+
+## Create a release PR
+
+1. Go to the [Release PR Workflow](https://github.com/open-telemetry/opentelemetry-js/actions/workflows/create-or-update-release.yml)
+2. Click "Run workflow"
+3. For `Release Type`, select if you want to create a release PR for a new `minor` or `patch` version.
+4. For `Release Scope`, select if you want to release
+   - `experimental` (all packages under `./experimental/packages`)
+   - `sdk` (all packages under `./packages/` and `./experimental/packages`)
+   - `all` (all packages under `./api/`, `./packages/` and `./experimental/packages`)
+
+> [!TIP]
+> If there was a commit to `main`, after PR creation simply run the workflow again before merging it.
+> Re-running it will update the PR with the contents from `main` and will update the PR body too.
+
+## Review and merge the release PR
+
+1. Review the PR generated via the workflow (it will be titled `chore: prepare next release` and opened by the @opentelemetrybot user)
+2. Once approved, merge the PR
+
+## Publish to NPM
+
+### Prerequisites
+
+1. Ensure you have access to the [`opentelemetry` npm organization](https://www.npmjs.com/org/opentelemetry)
```

Comment:
Do you know how I could request access to the npm "opentelemetry" org?

Reply:
Currently this org is owned by one of the people on the maintainers group (@dyladan), so asking them is the current way - I did not want to document it yet as I think we should transfer that ownership of the org to the TC via a maintenance user. The way to do it then would be to open an issue on the community repo.

Ideally I'd also like to have the releases published by a maintenance user when we start using a workflow to publish to npm.